### PR TITLE
Fat 3438 c11115

### PR DESCRIPTION
**Pull request by TetianaParanich**
***
https://issues.folio.org/browse/FAT-3438
***
https://foliotest.testrail.io//index.php?/cases/view/11115
***

### CODE REVIEW

File: cypress/e2e/data-import/field-protection-overrides.cy.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import SettingsMenu from '../../support/fragments/settingsMenu';
 import NewFieldMappingProfile from '../../support/fragments/data_import/mapping_profiles/newFieldMappingProfile';
 import FieldMappingProfiles from '../../support/fragments/data_import/mapping_profiles/fieldMappingProfiles';
 import MarcFieldProtection from '../../support/fragments/settings/dataImport/marcFieldProtection';
-import MappingProfileDetails from '../../support/fragments/data_import/mapping_profiles/mappingProfileDetails';
+import MappingProfileDetailsView from '../../support/fragments/data_import/mapping_profiles/mappingProfileDetailsView';
```
Comment (re-jas) [inline cypress/e2e/data-import/field-protection-overrides.cy.js:8]: Unable to resolve path to module '../../support/fragments/data_import/mapping_profiles/mappingProfileDetailsView'
I think it should be '../../support/fragments/data_import/mapping_profiles/fieldMappingProfileView'. Check please

File: cypress/e2e/data-import/make-fields-required-in-invoice-map-profile.cy.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ import FieldMappingProfiles from '../../support/fragments/data_import/mapping_pr
 import NewFieldMappingProfile from '../../support/fragments/data_import/mapping_profiles/newFieldMappingProfile';
 import Helper from '../../support/fragments/finance/financeHelper';
 import Users from '../../support/fragments/users/users';
-import MappingProfileDetails from '../../support/fragments/data_import/mapping_profiles/mappingProfileDetails';
+import MappingProfileDetailsView from '../../support/fragments/data_import/mapping_profiles/mappingProfileDetailsView';
```
Comment (re-jas) [inline cypress/e2e/data-import/make-fields-required-in-invoice-map-profile.cy.js:8]: Unable to resolve path to module '../../support/fragments/data_import/mapping_profiles/mappingProfileDetailsView'
I think it should be '../../support/fragments/data_import/mapping_profiles/fieldMappingProfileView'. Check please

File: cypress/e2e/data-import/field-protection-overrides.cy.js (hunk 1 above)
Comment (TetianaParanich) [reply cypress/e2e/data-import/field-protection-overrides.cy.js]: Done

File: cypress/e2e/data-import/make-fields-required-in-invoice-map-profile.cy.js (hunk 2 above)
Comment (TetianaParanich) [reply cypress/e2e/data-import/make-fields-required-in-invoice-map-profile.cy.js]: Done

**Review by MaksimTerekhov — APPROVED** [review]

**Review by re-jas — APPROVED** [review]

**Pull request merged**